salt/utils/sanitisers.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2016 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import os.path
+from builtins import str as text
```

Comment:
``builtins`` is not in the stdlib, and would require adding ``future`` (not to be confused with ``futures``) as a dependency.

If the aim is to enforce unicode on PY2, ``six.text_type`` already does this, and we distribute six so there's no need to add a dependency for it.

```python
>>> from salt.ext import six
>>> six.text_type('foo')
u'foo'
```